### PR TITLE
Start webxdc app when tapping anywhere in the cell

### PR DESCRIPTION
**Pull request by Amzd**
Start webxdc app when tapping anywhere in the cell, not just image and action button

### CODE REVIEW

**Comment by Simon-Laux** [comment]
On desktop you can select the text, so we shouldn't copy this to desktop if we decide to go for it. At least not for messages with text.
IMO we can also open up the design question again for the webxdc message, I had some suggestions back when we did that, but we had no time back then.
Like move the text that is sent with the webxdc down and the start button up and visually combine webxdc icon + webxdc title + webxdc document title and start button in a box, and then make that box tap-able?

**Comment by Amzd** [comment]
> is that expected as well? or just a coincidence that may change and strike back during maintenance later?

The messageLabel in the cell has a gestureRecognizer (see: `BaseMessageCell.swift:257`) which stops the tableView(_:didSelectRow:) from being called. Preferably the logic at `BaseMessageCell.swift:279` would be moved to didSelectRow because you also get a point there, and then there is no ambiguity.

Right now if you tap text inside the cell even if there is no url, it does not actually start the webxdc app because of that gestureRecognizer. I'll fix that.

File: deltachat-ios/Chat/ChatViewController.swift [hunk 1]
```diff
@@ -1124,7 +1122,6 @@ class ChatViewController: UITableViewController, UITableViewDropDelegate {
         messageInputBar.inputTextView.placeholder = String.localized("chat_input_placeholder")
         messageInputBar.inputTextView.accessibilityLabel = String.localized("write_message_desktop")
         messageInputBar.separatorLine.backgroundColor = DcColors.colorDisabled
-        messageInputBar.inputTextView.tintColor = DcColors.primary
```
[the remark this answers is not in the document]
Comment (Amzd) [reply deltachat-ios/Chat/ChatViewController.swift:1127]: Yep, that is a duplicate line, see few lines above.

**Comment by Amzd** [comment]
> Like move the text that is sent with the webxdc down and the start button up and visually combine webxdc icon + webxdc title + webxdc document title and start button in a box, and then make that box tap-able?

Love that. The start link is a bit out of style.

**Review by r10s — APPROVED** [review]
yeah, lgtm - and thanks for the explanations!

**Comment by adbenitez** [comment]
> @adbenitez i am sure we taked about making the whole cell tappable on android as well, but decided to not go for that - do you remember reasonings? or was it just that it was the simplest thing to do that time? (i am not suggesting to change android as well immediately, it is good if one OS goes ahead :)

by "cell" you mean the message bubble? to me this sounds like pretty weird behavior, if that is common in iOS it is fine I guess, but I don't think it makes much sense in Android, you have the button and the big app icon (similar to a launcher icon) I don't see why users would touch (potentially by accident) in the message text and that would open the app, sounds unnecessary and I have never seen this in apps in android

**Comment by adbenitez** [comment]
> Love that. The start link is a bit out of style.

+1000000000

**Pull request merged**